Documentation/document.html

```diff
@@ -238,7 +238,21 @@ <h1>Kontum</h1>
                           <iframe src="https://www.thiscodeworks.com/embed/6207a9d03d90130015398c14" style="width: 100%; height: 177px;" frameborder="0"></iframe>
 
                     </div>
+             
+            <p id="" class="doc-header">Snackbar</p>
+                    <div class="doc-avatar flex-center">
 
+                        <div class="snackbar flex-space-btw snackbar-bg-color">
+                            <div class="snackbar-bg-color"><i class="snackbar-icon fas fa-exclamation snackbar-bg-color"></i>Can't send message. Retry in 5 seconds.</div>
```

Comment:
if you have given a bg color to parent then why you require it in child, it will already work

Reply:
done